docs/customization/synchronization.md

```diff
@@ -1,9 +1,9 @@
-# Resource Hooks
+# Synchronization Customization
```

Comment:
I prefer to keep this named "Resource Hooks", since it is not intuitive to find this information under generic title such as "synchronization"